tensorrt_llm/bench/benchmark/throughput.py

```diff
@@ -373,6 +383,8 @@ def throughput_command(
         logger.info("Setting up throughput benchmark.")
         kwargs = kwargs | runtime_config.get_llm_args()
         kwargs['backend'] = backend
+        if stream_interval is not None:
```

Comment:
Might it be worth throwing an error when someone is trying to specify a stream interval without using `--streaming`?